relay-event-normalization/src/normalize/mod.rs

```diff
@@ -479,6 +480,71 @@ fn normalize_measurements(
     }
 }
 
+/// Computes performance score measurements
+///
+/// This computes score from vital measurements, using config options to define how it is
+/// calculated.
+fn normalize_performance_score(
+    event: &mut Event,
+    performance_score_config: Option<&PerformanceScoreConfig>,
+) {
+    if let Some(performance_score_config) = performance_score_config {
+        for profile in &performance_score_config.profiles {
+            if let Some(condition) = &profile.condition {
+                if !condition.matches(event) {
+                    continue;
+                }
```

Comment:
Should we `break` at the end of this loop iteration so we don't accidentally apply two profiles (first matching profile wins)?

Reply:
oh yeah we should, missed that, thanks 👍